### ISSUE TITLE
[Bug] For web accessibility, side menu items should be links, not buttons

### ISSUE THREAD

**Issue by kgryte** [stdlib-js/www#12]
## Checklist

> Please ensure the following tasks are completed before filing a bug report.

-   [x] Read and understood the [Code of Conduct][code-of-conduct].
-   [x] Searched for existing issues and pull requests.

## Description

> Description of the issue.

Currently, the docs side menu renders navigation elements as `<button>` elements, rather than `<links>`. This breaks user expectations regarding behavior (i.e., navigating to new content) and ability to right-click and, e.g., open in a new tab or window.

Preferably, the markup would be refactored to use anchor `<a>` tags, rather than `<button>` tags.

## Related Issues

> Does this issue have any related issues?

No.

## Questions

> Any questions for reviewers?

No.

## Other

> Any other information relevant to this issue? This may include screenshots, references, stack traces, sample output, and/or implementation notes.

#### Demo

> If relevant, provide a link to a live demo.

For a live demo of the issue, see

-   navigate to `/docs/api`.
-   right-click on a side menu item.
-   open the console/inspector.
-   note the use of `button`.

#### Reproduction

> What steps are required to reproduce the unexpected output?

In order to reproduce this bug, do the following:

-   see above.

#### Expected Results

> What are the expected results?

The following results are expected:

-   I would expect `<a>` tags.

#### Actual Results

> What are the actual results?

The following are the actual results:

-   the use of `<button>` tags.

#### Environments

> What environments are affected (e.g., `Node v0.4.x`, `Chrome`, `IE 11`)? If Node.js, include the `npm` version, operating system, and any other potentially relevant platform information.

The following environments are affected:

-   all browsers.

[code-of-conduct]: https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md

**Comment by kgryte** [comment]
This has now been addressed. Side menu items are now links, rather than buttons.
